backup/moodle2/restore_turnitintooltwo_stepslib.php

```diff
@@ -154,8 +154,18 @@ protected function process_turnitintooltwo_submissions($data) {
             $DB->insert_record('turnitintooltwo_users', $tiiuser);
         }
 
-        $newitemid = $DB->insert_record('turnitintooltwo_submissions', $data);
-        $this->set_mapping('turnitintooltwo_submissions', $oldid, $newitemid, true);
+        // Check if this hash already exists.
+        if ($check_hash = $DB->get_record('turnitintooltwo_submissions', array('submission_hash' => $data->submission_hash))) {
+            // Update the row/grade if this submission is the most recent submission.
+            if ($check_hash->submission_modified >= $data->submission_modified) {
```

Comment:
If this is greater than or equal to, wouldn't that mean that every time we call `process_turnitintooltwo_submissions` we'll overwrite all of the papers even if the modified time is the same? That would add a lot of DB updates for no reason.